src/components/Home/Home.jsx

```diff
@@ -1,27 +1,73 @@
 import React from 'react';
-import { Redirect, useHistory } from 'react-router-dom';
+import { useState } from 'react';
+import { useForm } from '../../hooks/useForm';
+import { useFirebase } from '../../hooks/useFirebase.js';
+import { useHistory } from 'react-router-dom';
 import getToken from '../../lib/tokens';
 
 const Home = () => {
-  const userToken = localStorage.getItem('token');
+  const [values, handleInputChange, setValues] = useForm({
+    token: '',
+  });
+
+  const [msn, setMsn] = useState('');
+  const [error, setError] = useState('');
```

Comment:
just if you want, what do think to create a custom hook, where you can handle error and loading and then you could import here or any location you want :D

Reply:
@dupjpr created the useNotification hook! 🥳🥳🥳